### PR TITLE
python310Packages.superqt: 0.3.3 -> 0.3.5

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/superqt/default.nix [hunk 1]
```diff
@@ -7,23 +7,34 @@
...
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-Ns3AFUL0BReIwTHfrlfXr/2GLtLvT7hfSjjh+r7btcY=";
+    sha256 = "sha256-nKNFV/mzdugQ+UJ/qB0SkCSm5vEpvI/tgHYKJr6NEyg=";
   };
   format = "pyproject";
+
   nativeBuildInputs = [ setuptools-scm ];
-  propagatedBuildInputs = [ pyqt5 qtpy typing-extensions ];
-  checkInputs = [ pytestCheckHook pytest ];
```
Comment (SomeoneSerge) [inline pkgs/development/python-modules/superqt/default.nix:25]: Maybe we keep `pytestCheckHook`, that'd remind us to occasionally test if `doCheck = true` still segfaults

File: pkgs/development/python-modules/superqt/default.nix [hunk 2]
```diff
@@ -7,23 +7,34 @@
...
+  version = "0.3.5";
 
   src = fetchFromGitHub {
     owner = "napari";
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-Ns3AFUL0BReIwTHfrlfXr/2GLtLvT7hfSjjh+r7btcY=";
+    sha256 = "sha256-nKNFV/mzdugQ+UJ/qB0SkCSm5vEpvI/tgHYKJr6NEyg=";
   };
   format = "pyproject";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/superqt/default.nix:23]:
```suggestion
```

File: pkgs/development/python-modules/superqt/default.nix [hunk 3]
```diff
@@ -7,23 +7,34 @@
 , typing-extensions
 , pytest
 , pytestCheckHook
+, pygments
 }:
 
 buildPythonPackage rec {
   pname = "superqt";
-  version = "0.3.3";
+  version = "0.3.5";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/superqt/default.nix:16]:
```suggestion
  format = "pyproject";

```

File: pkgs/development/python-modules/superqt/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/superqt/default.nix:25]: sure, why not

**Comment by onny** [comment]
Thank you both for reviewing :)

**Pull request merged**